hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractEvictableRecordStore.java

```diff
@@ -300,6 +305,14 @@ public void doPostEvictionOperations(Record record, boolean backup) {
         }
     }
 
+    private void invalidateNearCache(Data key) {
+        if (!mapContainer.hasInvalidationListener()) {
+            return;
+        }
+
+        invalidator.invalidateKey(key, name, "NA");
```

Comment:
What is this `sourceUUID` used for? Can you use `key.toString()` or `name` instead?

Reply:
Or can you use `nodeEngine.getLocalMember().getUuid()`?

Reply:
Added this explanation: `sourceUUID` is used not to apply one invalidation twice to the original caller. But, in this case, we shouldn't use any valid source-uuid since invalidations are generated by the expiration system and all near-caches should be invalidated.